src/app/common-ui/Loading/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styles from './style.scss';
+
+import Spinner from '../../../assets/images/spinner.gif';
```

Comment:
I have added the alias `src`, you can rewrite the path using `src`.

```suggestion
import Spinner from 'src/assets/images/spinner.gif';
```